TRENZ.Lib.RazorMail.Core/Models/MailImportance.cs

```diff
@@ -0,0 +1,34 @@
+namespace TRENZ.Lib.RazorMail.Models;
+
+/// <summary>
+/// Specifies the urgency in which the mail should be delivered.
+/// </summary>
+/// <remarks>
+/// This sets both the <c>Importance</c> and <c>X-Priority</c> headers, for
+/// broad compatibility.
+/// </remarks>
+public enum MailImportance
```

Comment:
Can you link some documentation for where to look up these values?

Reply:
Done!

https://www.rfc-editor.org/rfc/rfc2156 (search for `importance`) and https://learn.microsoft.com/en-us/openspecs/exchange_server_protocols/ms-oxcmail/2bb19f1b-b35e-4966-b1cb-1afd044e83ab (the bottom table for `X-Priority`)

Reply:
nice